.github/workflows/deploy-to-control-plane.yml

```
@@ -182,45 +198,27 @@
-          (steps.get-pr.outputs.has_pr == 'true' && steps.check-app.outputs.app_exists == 'true')
-        run: git checkout ${{ steps.getRef.outputs.PR_REF }}
+        if: env.IS_STAGING != 'true'
+        run: git checkout ${{ steps.setup.outputs.PR_REF }}
 
       - name: Build Docker Image
-        if: |
-          (github.event_name == 'issue_comment' && 
-           github.event.issue.pull_request && 
-           github.event.comment.body == '/deploy-review-app') || 
-          (steps.get-pr.outputs.has_pr == 'true' && steps.check-app.outputs.app_exists == 'true')
         uses: ./.github/actions/build-docker-image
```

actionlint (1.7.4)

missing input "PR_NUMBER" which is required by action "Build Docker Image" defined at "./.github/actions/build-docker-image". all required inputs are "PR_NUMBER", "app_name", "commit", "org"

(action)